kuma/wiki/tests/conftest.py

```diff
@@ -185,6 +185,103 @@ def doc_hierarchy(wiki_user):
 
 
 @pytest.fixture
+def doc_hierarchy_with_zones(settings, doc_hierarchy, wiki_user_2, wiki_user_3):
+    top_doc = doc_hierarchy.top
+    it_top_doc = Document.objects.create(
+        locale='it',
+        slug='superiore',
+        title='Superiore Documento',
+        rendered_html='<p>Superiore...</p>',
+        parent=top_doc
+    )
+    Revision.objects.create(
+        document=it_top_doc,
+        creator=wiki_user_2,
+        based_on=top_doc.current_revision,
+        content='<p>Superiore...</p>',
+        title='Superiore Documento',
+        created=datetime(2017, 4, 30, 11, 17)
+    )
+    de_top_doc = Document.objects.create(
+        locale='de',
+        slug='oben',
+        title='Oben Dokument',
+        rendered_html='<p>Oben...</p>',
+        parent=top_doc
+    )
+    Revision.objects.create(
+        document=de_top_doc,
+        creator=wiki_user_2,
+        based_on=top_doc.current_revision,
+        content='<p>Oben...</p>',
+        title='Oben Dokument',
+        created=datetime(2017, 4, 30, 10, 03)
```

Comment:
I agree w/ GitHub highlighting - ``3`` not ``03``, which means '3 in octal' (08, 09 don't work)